docs/api-inference/parameters.md

```diff
@@ -21,25 +21,25 @@ To do this, you can add `x-use-cache:false` to the request headers. For example
 <inferencesnippet>
 
 <curl>
-```bash
+```diff
 curl https://api-inference.huggingface.co/models/MODEL_ID \
     -X POST \
     -d '{"inputs": "Can you please let us know more details about your "}' \
     -H "Authorization: Bearer hf_***" \
     -H "Content-Type: application/json" \
-    -H "x-use-cache: false"
++   -H "x-use-cache: false"
```

Comment:
Does this work well? I tried in local deploy and this didn't render well

Reply:
@osanseviero highlighting doesn't work locally but it works as expected when deployed. I've opened a dummy PR  (https://github.com/huggingface/huggingface_hub/pull/2492) to test it works: https://moon-ci-docs.huggingface.co/docs/huggingface_hub/pr_2492/en/index?code=python.

![image](https://github.com/user-attachments/assets/235324c3-2e77-4a49-bc4f-2e0399e2c773)

![image](https://github.com/user-attachments/assets/3b580f88-1218-4119-a18e-d18e96deecd2)


![image](https://github.com/user-attachments/assets/a8c54569-9e90-4b02-9cc4-2a89156439eb)